go.mod

```diff
@@ -19,9 +19,10 @@ require (
 	github.com/benbjohnson/clock v1.3.0
 	github.com/bradleyjkemp/cupaloy v2.3.0+incompatible
 	github.com/elastic/go-elasticsearch/v7 v7.17.1
-	github.com/estuary/flow v0.2.4-0.20230623123341-b5c714e822e3
+	github.com/estuary/flow v0.3.4-0.20230714153247-33844329ca87
 	github.com/evanphx/json-patch/v5 v5.6.0
 	github.com/go-mysql-org/go-mysql v1.5.0
+	github.com/go-openapi/jsonpointer v0.19.6
```

Comment:
I don't think we still need all of these additions to `go.mod`, so it would be best to revert them unless they are needed.